### PR TITLE
ViewMyProfile

### PR DESCRIPTION
**Pull request by mincho-cb-nytd**

### CODE REVIEW

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'CB User NYTD');
+    viewMyProfile.elements.username().should('have.text', 'nytdcb')
+    viewMyProfile.elements.phoneNumber().should('have.text', '(508) 246-0315')
+    viewMyProfile.elements.email().should('have.text', 'mincho.rusev@icf.com')
+    viewMyProfile.elements.receivesEmailNotification().should('have.text', 'Receives emails')
+    viewMyProfile.elements.userType().should('have.text', 'Central Office User')
+    viewMyProfile.elements.dataExportText().should('have.text'," Data Export")
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
+    viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
+    viewMyProfile.elements.returnToTopBtn().should('have.text', 'Return to top').click();
```
Comment (wyntonjones) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:22]: line 22, what does this click do? you don't verify anything after action.

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import ViewMyProfile from "../pages/MyProfilePageObjects";
+const viewMyProfile = new ViewMyProfile();
+describe("State Authorized Official Account Request Validations", function () {
+    beforeEach(() => {
+        cy.login('nytdcb', 'P@ssw0rd1') // Login with session, implemented in commands.js
+    })
+    it("Verify CB central users are able to view my profile user information", function () {
+    cy.visit('/User');
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
```
Comment (wyntonjones) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:12]: add url assertion statements. after selecting my profile, verify page url is correct

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'CB User NYTD');
+    viewMyProfile.elements.username().should('have.text', 'nytdcb')
+    viewMyProfile.elements.phoneNumber().should('have.text', '(508) 246-0315')
+    viewMyProfile.elements.email().should('have.text', 'mincho.rusev@icf.com')
+    viewMyProfile.elements.receivesEmailNotification().should('have.text', 'Receives emails')
+    viewMyProfile.elements.userType().should('have.text', 'Central Office User')
+    viewMyProfile.elements.dataExportText().should('have.text'," Data Export")
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (wyntonjones) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:20]: line 20, add a click to this statement and then add an assertion of the edit my profile url. then back to view my profile.

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (wyntonjones) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:21]: line 21, click button and verify delete modal text and fields. but do not delete

File: cypress/e2e/My Profile/TestStateUser.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    viewMyProfile.elements.userType().should('have.text', 'Test State')
+    viewMyProfile.elements.requestStateManagerAccessBtnForTestStateUser().should('have.text',"Request State Manager Access")
+    viewMyProfile.elements.requestStateAuthOfficialAccessBtnForSTestStateUser().should('have.text', "Request State Authorized Official Access")
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
+    viewMyProfile.elements.updatePasswordBtn().should('have.text','Update Password')
+    viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
+    viewMyProfile.elements.returnToTopBtn().should('have.text', 'Return to top').click();
+    });
+
+    it("Verify test state users are able to view my pofile via welcome username ", function () {
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/TestStateUser.js:26]: Spelling - "Verify test state users are able to view my profile via welcome username"

File: cypress/e2e/My Profile/TestStateUser.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import ViewMyProfile from "../pages/MyProfilePageObjects";
+const viewMyProfile = new ViewMyProfile();
+describe("State Authorized Official Account Request Validations", function () {
+    beforeEach(() => {
+        cy.login('teststateuser', 'P@ssw0rd1') // Login with session, implemented in commands.js
+    })
+    it("Verify test state users are able to view my profile user information", function () {
+    cy.visit('/User');
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/TestStateUser.js:12]: Verify URL and verify bread crumb

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    viewMyProfile.elements.returnToTopBtn().should('have.text', 'Return to top').click();
+    });
+
+    it("Verify CB central users are able to view my profile via Welcome username", function () {
```
Comment (wyntonjones) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:25]: apply suggestions above, here

File: cypress/e2e/My Profile/SystemAdminUserMyProfile.js [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+import ViewMyProfile from "../pages/MyProfilePageObjects";
+const viewMyProfile = new ViewMyProfile();
+describe("State Authorized Official Account Request Validations", function () {
+    beforeEach(() => {
+        cy.login('nytdsysadmin', 'P@ssw0rd1') // Login with session, implemented in commands.js
+    })
+    it("Verify System admin users are able to view my profile user information", function () {
+    cy.visit('/User');
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/SystemAdminUserMyProfile.js:12]: Verify URL and verify bread crumb

File: cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+import ViewMyProfile from "../pages/MyProfilePageObjects";
+const viewMyProfile = new ViewMyProfile();
+describe("State Authorized Official Account Request Validations", function () {
+    beforeEach(() => {
+        cy.login('teststatesao', 'P@ssw0rd1') // Login with session, implemented in commands.js
+    })
+    it("Verify state authorization official  users are able to view my profile user information", function () {
+    cy.visit('/User');
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js:12]: Verify URL and verify bread crumb

File: cypress/e2e/My Profile/NytdRegionalUserMyProfile.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+import ViewMyProfile from "../pages/MyProfilePageObjects";
+const viewMyProfile = new ViewMyProfile();
+describe("My profile validation", function () {
+    beforeEach(() => {
+        cy.login('nytdregional', 'P@ssw0rd1') // Login with session, implemented in commands.js
+    })
+    it("Verify CB central users are able to view my profile user information", function () {
+    cy.visit('/User');
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/NytdRegionalUserMyProfile.js:12]: Verify URL and verify bread crumb

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js (hunk 2 above)
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:12]: Verify URL and verify bread crumb

File: cypress/e2e/My Profile/NytdRegionalUserMyProfile.js
Comment (wyntonjones) [inline cypress/e2e/My Profile/NytdRegionalUserMyProfile.js:1]: Apply comments made in CBCentralUserMyProfile test.

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+        cy.visit('/User');
+        commonObjects.clickOnWelcomeBtn();
+        viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'CB User NYTD');
+        viewMyProfile.elements.username().should('have.text', 'nytdcb')
+        viewMyProfile.elements.phoneNumber().should('have.text', '(508) 246-0315')
+        viewMyProfile.elements.email().should('have.text', 'mincho.rusev@icf.com')
+        viewMyProfile.elements.receivesEmailNotification().should('have.text', 'Receives emails')
+        viewMyProfile.elements.userType().should('have.text', 'Central Office User')
+        viewMyProfile.elements.dataExportText().should('have.text', " Data Export")
+        viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:35]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+        viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:36]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/NytdRegionalUserMyProfile.js [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+        cy.visit('/User');
+        commonObjects.clickOnWelcomeBtn();
+        viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'Regional User NYTD');
+        viewMyProfile.elements.username().should('have.text', 'nytdregional')
+        viewMyProfile.elements.emailForRegionalID().should('have.text', 'wynton.jones@icf.com')
+        viewMyProfile.elements.receivesEmailNotificationForRegional().should('have.text', 'Receives emails')
+        viewMyProfile.elements.region().should('have.text', 'Region 11')
+        viewMyProfile.elements.assignedStatesText().should('have.text', 'Assigned States')
+        viewMyProfile.elements.assignedStates().should('have.text', 'Test State')
+        viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/NytdRegionalUserMyProfile.js:35]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/NytdRegionalUserMyProfile.js [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+        viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/NytdRegionalUserMyProfile.js:36]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+        cy.visit('/User');
+        commonObjects.clickOnWelcomeBtn();
+        viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'teststate sao');
+        viewMyProfile.elements.username().should('have.text', 'teststatesao')
+        viewMyProfile.elements.emailForTestStateSao().should('have.text', 'teststatesao@test.com')
+        viewMyProfile.elements.receivesEmailNotificationForTestStatesao().should('have.text', 'Receives emails')
+        viewMyProfile.elements.userType().should('have.text', 'Test State')
+        viewMyProfile.elements.userTypeForSAO().should('have.text', 'State Authorized Official')
+        viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
+        viewMyProfile.elements.updatePasswordBtn().should('have.text', 'Update Password')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js:34]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+    it("Verify state authorization official  users are able to view my profile vie Welcome username", function () {
+        cy.visit('/User');
+        commonObjects.clickOnWelcomeBtn();
+        viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'teststate sao');
+        viewMyProfile.elements.username().should('have.text', 'teststatesao')
+        viewMyProfile.elements.emailForTestStateSao().should('have.text', 'teststatesao@test.com')
+        viewMyProfile.elements.receivesEmailNotificationForTestStatesao().should('have.text', 'Receives emails')
+        viewMyProfile.elements.userType().should('have.text', 'Test State')
+        viewMyProfile.elements.userTypeForSAO().should('have.text', 'State Authorized Official')
+        viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js:33]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+        viewMyProfile.elements.updatePasswordBtn().should('have.text', 'Update Password')
+        viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js:35]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'teststate mgr');
+    viewMyProfile.elements.username().should('have.text', 'teststatemgr')
+    viewMyProfile.elements.emailForTestStateMngID().should('have.text', 'wyntonj1@yahoo.com')
+    viewMyProfile.elements.receivesEmailNotificationForTestStageMng().should('have.text', 'Receives emails')
+    viewMyProfile.elements.userType().should('have.text', 'Test State')
+    viewMyProfile.elements.userTypeForTestStateMng().should('have.text', 'Manager')
+    viewMyProfile.elements.requestStateAuthOfficialAccessBtn().should('have.text',"Request State Authorized Official Access")
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:20]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+    viewMyProfile.elements.updatePasswordBtn().should('have.text','Update Password')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:21]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+    viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:22]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js [hunk 21]
```diff
@@ -0,0 +1,38 @@
+import CommonPageObjects from "../pages/CommonPageObjects";
+const commonObjects = new CommonPageObjects();
+import ViewMyProfile from "../pages/MyProfilePageObjects";
+const viewMyProfile = new ViewMyProfile();
+describe("State Authorized Official Account Request Validations", function () {
+    beforeEach(() => {
+        cy.login('teststatesao', 'P@ssw0rd1') // Login with session, implemented in commands.js
+    })
+    it("Verify state authorization official  users are able to view my profile user information", function () {
```
Comment (wyntonjones) [inline cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js:9]: line 9, "State Authorized Official or SAO"

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js (hunk 3 above)
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:20]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js (hunk 4 above)
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:21]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js (hunk 1 above)
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:22]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/CBCentralUserMyProfile.js [hunk 22]
```diff
@@ -0,0 +1,39 @@
...
+        viewMyProfile.elements.returnToTopBtn().should('have.text', 'Return to top').click();
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/CBCentralUserMyProfile.js:37]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js
Comment (wyntonjones) [inline cypress/e2e/My Profile/StateAutorizationOfficialUserMyProfile.js:1]: Apply comments made in CBCentralUserMyProfile test. Rename file to StateAuthorizedOfficialUserMyProfile

File: cypress/e2e/My Profile/NytdRegionalUserMyProfile.js [hunk 23]
```diff
@@ -0,0 +1,39 @@
...
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'Regional User NYTD');
+    viewMyProfile.elements.username().should('have.text', 'nytdregional')
+    viewMyProfile.elements.emailForRegionalID().should('have.text', 'wynton.jones@icf.com')
+    viewMyProfile.elements.receivesEmailNotificationForRegional().should('have.text', 'Receives emails')
+    viewMyProfile.elements.region().should('have.text', 'Region 11')
+    viewMyProfile.elements.assignedStatesText().should('have.text', 'Assigned States')
+    viewMyProfile.elements.assignedStates().should('have.text', 'Test State')
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/NytdRegionalUserMyProfile.js:20]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/NytdRegionalUserMyProfile.js [hunk 24]
```diff
@@ -0,0 +1,39 @@
...
+    viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
+    viewMyProfile.elements.returnToTopBtn().should('have.text', 'Return to top').click();
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/NytdRegionalUserMyProfile.js:22]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js [hunk 25]
```diff
@@ -0,0 +1,41 @@
...
+    cy.visit('/User');
+    commonObjects.clickOnAccountSettingsDropdown();
+    commonObjects.clickOnMyProfileSelect();
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'teststate mgr');
+    viewMyProfile.elements.username().should('have.text', 'teststatemgr')
+    viewMyProfile.elements.emailForTestStateMngID().should('have.text', 'wyntonj1@yahoo.com')
+    viewMyProfile.elements.receivesEmailNotificationForTestStageMng().should('have.text', 'Receives emails')
+    viewMyProfile.elements.userType().should('have.text', 'Test State')
+    viewMyProfile.elements.userTypeForTestStateMng().should('have.text', 'Manager')
+    viewMyProfile.elements.requestStateAuthOfficialAccessBtn().should('have.text',"Request State Authorized Official Access")
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:19]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js [hunk 26]
```diff
@@ -0,0 +1,41 @@
...
+    it("Verify state manager users are able to view my profile via welcome button", function () {
+        cy.visit('/User');
+        commonObjects.clickOnWelcomeBtn();
+        viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'teststate mgr');
+        viewMyProfile.elements.username().should('have.text', 'teststatemgr')
+        viewMyProfile.elements.emailForTestStateMngID().should('have.text', 'wyntonj1@yahoo.com')
+        viewMyProfile.elements.receivesEmailNotificationForTestStageMng().should('have.text', 'Receives emails')
+        viewMyProfile.elements.userType().should('have.text', 'Test State')
+        viewMyProfile.elements.userTypeForTestStateMng().should('have.text', 'Manager')
+        viewMyProfile.elements.requestStateAuthOfficialAccessBtn().should('have.text', "Request State Authorized Official Access")
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:35]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js [hunk 27]
```diff
@@ -0,0 +1,41 @@
...
+        viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:36]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/SystemAdminUserMyProfile.js [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text','nytd sysadmin')
+    viewMyProfile.elements.username().should('have.text', 'nytdsysadmin')
+    viewMyProfile.elements.phoneNumber().should('have.text', 'ZAP')
+    viewMyProfile.elements.email().should('have.text', 'wynton.jones@icf.com')
+    viewMyProfile.elements.receivesEmailNotification().should('have.text', 'Receives emails')
+    viewMyProfile.elements.userType().should('have.text', 'System Administrator')
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/SystemAdminUserMyProfile.js:19]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/SystemAdminUserMyProfile.js [hunk 29]
```diff
@@ -0,0 +1,37 @@
...
+    it("Verify System admin users are able to view my profile via welcome username", function () {
+        cy.visit('/User');
+        commonObjects.clickOnWelcomeBtn();
+        viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'nytd sysadmin')
+        viewMyProfile.elements.username().should('have.text', 'nytdsysadmin')
+        viewMyProfile.elements.phoneNumber().should('have.text', 'ZAP')
+        viewMyProfile.elements.email().should('have.text', 'wynton.jones@icf.com')
+        viewMyProfile.elements.receivesEmailNotification().should('have.text', 'Receives emails')
+        viewMyProfile.elements.userType().should('have.text', 'System Administrator')
+        viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/SystemAdminUserMyProfile.js:33]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/SystemAdminUserMyProfile.js [hunk 30]
```diff
@@ -0,0 +1,37 @@
...
+        viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/SystemAdminUserMyProfile.js:34]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/StateManagerUserMyProfile.js
Comment (wyntonjones) [inline cypress/e2e/My Profile/StateManagerUserMyProfile.js:1]: Apply comments made in CBCentralUserMyProfile test.

File: cypress/e2e/My Profile/TestStateUser.js [hunk 31]
```diff
@@ -0,0 +1,43 @@
...
+    viewMyProfile.elements.myProfileFirstLastName().should('have.text', 'teststate user');
+    viewMyProfile.elements.username().should('have.text', 'teststateuser')
+    viewMyProfile.elements.emailForTestStateUser().should('have.text', 'teststateuser@test.com')
+    viewMyProfile.elements.receivesEmailNotificationForTestState().should('have.text', 'Does not receive emails')
+    viewMyProfile.elements.userType().should('have.text', 'Test State')
+    viewMyProfile.elements.requestStateManagerAccessBtnForTestStateUser().should('have.text',"Request State Manager Access")
+    viewMyProfile.elements.requestStateAuthOfficialAccessBtnForSTestStateUser().should('have.text', "Request State Authorized Official Access")
+    viewMyProfile.elements.editMyProfileBtn().should('have.text', 'Edit My Profile')
+    viewMyProfile.elements.updatePasswordBtn().should('have.text','Update Password')
+    viewMyProfile.elements.deleteMyAccountBtn().should('have.text', 'Delete My Account')
```
Comment (TylerFSmith2) [inline cypress/e2e/My Profile/TestStateUser.js:22]: Verify functionality of button in a separate test

File: cypress/e2e/My Profile/SystemAdminUserMyProfile.js
Comment (wyntonjones) [inline cypress/e2e/My Profile/SystemAdminUserMyProfile.js:1]: Apply comments made in CBCentralUserMyProfile test.

File: cypress/e2e/My Profile/TestStateUser.js
Comment (wyntonjones) [inline cypress/e2e/My Profile/TestStateUser.js:1]: Apply comments made in CBCentralUserMyProfile test. FIle name is not consistent with other files. Acceptance criteria you call this the Default State User, so use the same term.

**Review by wyntonjones — CHANGES_REQUESTED** [review]

**Comment by wyntonjones** [comment]
In addition to the previous comments, add tests to verify the Regional Office POC that is shown for state user my profiles.

**Pull request merged**